CONTRIBUTING.md

```diff
@@ -188,3 +258,45 @@ for a reference on project code style and best practices.
 
 For standards on contributing to documentation, see the
 [Documentation guidelines](http://opencolorio.org/developers/documentation_guidelines.html).
+
+## Test Policy
+
+All functionality in OpenColorIO must be covered by an automated test. Tests
+should be implemented in a separate source file under the ``tests`` project
```

Comment:
Is "project" the correct term?  Also on line 267 you have a spurious "the" before OCIO.

Reply:
Updated.